### PR TITLE
IdentityBadge + AddressField: use a monospace font

### PR DESCRIPTION
**Pull request by bpierre**
<p align=center>

</p>

Note: the monospace font is only used if an address (shortened or not) is being displayed, not for custom labels.

### CODE REVIEW

File: src/components/Text/README.md [hunk 1]
```diff
@@ -28,6 +28,17 @@ Set this property to a color value to change the text color.
...
 
+### `monospace`
+
+- Type: `Boolean`
+- Default value: `false`
+
+Set this property to a `true` to render the text using a monospace font.
+
+```jsx
+<Text smallcaps>Title Example</Text>
```
Comment (AquiGorka) [inline src/components/Text/README.md]: In this example, should you uses the `monospace` prop?

Comment (bpierre) [reply src/components/Text/README.md]: Yes!

File: src/utils/styles/font.js [hunk 2]
```diff
@@ -38,10 +38,23 @@ const smallcapsCss = smallcaps =>
...
+  monospace = false,
+}) => {
   return `
-    ${fontSizeCss(size, inherit)};
-    ${weightCss(weight, inherit)};
-    ${smallcapsCss(smallcaps, inherit)};
+    ${fontSizeCss(size)};
+    ${weightCss(weight)};
+    ${smallcapsCss(smallcaps)};
+    ${monospaceCss(monospace)};
```
Comment (sohkai) [inline src/utils/styles/font.js]: Nice, we must've forgot to take out the `inherit` earlier!

Comment (bpierre) [reply src/utils/styles/font.js]: Yes 😆

**Pull request merged**